lib/mongoid/serializable.rb

```diff
@@ -13,6 +13,8 @@ module Serializable
     included do
 
       class << self
+         remove_method :include_root_in_json if method_defined?(:include_root_in_json)
+         remove_method :include_root_in_json= if method_defined?(:include_root_in_json=)
```

Comment:
Nice! I had to dig a bit to understand that these methods are defined in ActiveModel, and we're just re-defining them here to account for our default value `Mongoid.include_root_in_json`. Maybe add a comment to that effect?

Also, there's an extra space before these lines -- we've standardized on indentations in 2-space increments.